scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SubmitEditServlet.java

```diff
@@ -246,4 +251,16 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
         }
     }
 
+    private String getTaskName(Node node) {
```

Comment:
I don't know well how to do it, as the context is completely different (client-side javascript generated by gwt / server-side servlets). But I will have a look